contents/teams/feature-flags/objectives.mdx

```diff
@@ -1,53 +1,28 @@
-## Q2 2025 Objectives
+### Q3 2025 Objectives
 
-### Flag Stability, Consistency, and Resiliency
-*Goal: Create a highly stable, consistent, and resilient feature flag system capable of supporting 10x growth*
+#### Objective 1: Behavioral Cohorts
 
-- Scale `/flags` service
-  - Complete 100% worldwide rollout of `/flags` endpoint to all customers
-  - Measure and validate improved query performance metrics
-  - Document infrastructure cost savings achieved
-  - Stretch: Investigate feasibility of decoupling flag evaluation from Postgres entirely
+Motivation: Move beyond static person properties to enable dynamic behavioral targeting – let customers target users based on what they've done (or haven't done) recently.
 
-- Standardize SDK Error Handling and UX
-  - Move more SDK functionality to the server wherever possible ([reference](https://github.com/PostHog/posthog/issues/30466))
-  - Implement unified error handling framework across all SDKs
-  - Create consistent fallback and default behaviors across platforms
-  - Ensure all SDKs utilize the latest (v2) flags response format
+What we will ship:
+* **Behavioral targeting engine** – Ship the ability to target users based on recent actions, like "users who haven't seen the pricing page in 7 days" or "users who completed onboarding but haven't upgraded" <TeamMember name="Dylan Martin" photo />
+* **Customer validation** – Partner with 5-10 key customers to validate use cases, gather feedback, and measure impact on their conversion and engagement <TeamMember name="Dylan Martin" photo /> and <TeamMember name="Annika Schmid" photo />
```

Comment:
@annikaschmid you cool sharing this with me?  I have some customers in mind but once we start getting this rolling I think having a hand-picked beta group to start with is the move.